server/controllers/accommodationController.js

```diff
@@ -0,0 +1,107 @@
+import AccommodationHelper from '../helpers/accommodationHelper';
+
+/**
+ * This class contains all methods
+ * required to handle all
+ * accommodation-related operations.
+ */
+class Accommodation {
+  /**
+   * This method handles the register request.
+   * @param {object} req The user's request.
+   * @param {object} res The response.
+   * @returns {object} The status and some data of the user.
```

Comment:
It is not some user data please, It is created accommodation instead. Be explicit.

Reply:
Sorry about that!
Feedback implemented.